tests/e2e/fixtures/kserve_denpendencies.py

```diff
@@ -0,0 +1,186 @@
+import os
+import logging
+import re
+import subprocess
+import time
+import pytest
+from e2e.utils.constants import TENSORFLOW_SERVING_VERSION
+from e2e.utils.s3_for_kserve.data_bucket import S3BucketForKServe
+from e2e.fixtures.cluster import cluster, create_iam_service_account, associate_iam_oidc_provider, delete_iam_service_account
+from e2e.utils.utils import rand_name, write_yaml_file, exec_shell, wait_for, load_yaml_file, kubectl_apply
+from e2e.utils.config import configure_resource_fixture, metadata 
+from e2e.conftest import region
+from e2e.utils.load_balancer.setup_load_balancer import create_certificates
+from e2e.utils.custom_resources import get_inference_service
+
+
+
+RANDOM_PREFIX = rand_name("kserve-")
+SECRET_CONFIG_FILE = "./resources/kserve/kserve-secret.yaml"
+INFERENCE_CONFIG_FILE = "./resources/kserve/inference-service.yaml"
+AUTHORIZATION_POLICY_CONFIG_FILE = "./resources/kserve/allow-predictor.yaml"
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope="class")
+def clone_tensorflow_serving():
```

Comment:
Would it be better to just host this small directory in our test directory instead of downloading it everytime?

Reply:
@surajkota

Reply:
maybe we can create an S3 bucket for KF tests similar to ACK tests since we are downloading dataset for testing SM components as well, ok for now